README.md

```diff
@@ -153,6 +153,8 @@ Short description.
 
   App that calculates the climate impact of your daily choices by connecting to apps and services you already use: your personal and automated carbon tracker.
 
+- [Climate Choice](https://climatechoice.co/) - A non-profit project that contains actionable guides on reducing climate change, as well as educational information about climate change.
```

Comment:
My bad, it's probably better to keep it on resources. I tried to search where they're founded, and realized just now that it's a project and not an organization. Sorry for the unnecessary change request :pray:

Reply:
No problem, reverted it.